### PR TITLE
feat: added snackbar component

### PR DESCRIPTION
**Pull request by kedark152**
### added snackbar component
1. added two snack-bars baseline & leading with cancel & retry buttons.
2. Displayed the Live Demo of snack-bar component

**Preview Link:** https://deploy-preview-8--forest-ui.netlify.app/docs-site/docs

### CODE REVIEW

**Review by KanekarSnehal — APPROVED** [review]
Good work. No issues found, only suggestion is your css code is not formatted, use consistent spacing between two code blocks.

File: stylesheets/components/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
+.btn-snackbar{
+    color: var(--white);
+    background-color: var(--primary-hover);
+    padding: 5px 8px;
```
Comment (sameep-s) [inline stylesheets/components/snackbar.css:4]: you can use rem units for consistency.

File: stylesheets/components/snackbar.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    bottom: 10%;
+    left: 15%;
+}
+
+
+/* Snackbar Static Demo to show on Docs Site */
+.snackbar-baseline-demo{
+    position: relative;
+}
+.snackbar-leading-demo{
```
Comment (sameep-s) [inline stylesheets/components/snackbar.css]: .snackbar-leading-demo,
.snackbar-baseline-demo{
    position: relative;
}

you can do something like that as they are having identical properties,

**Review by sameep-s — APPROVED** [review]
Code is good, just some suggestions added.

File: stylesheets/components/snackbar.css (hunk 1 above)
Comment (kedark152) [reply stylesheets/components/snackbar.css:4]: Thanks for your review!! We can use px for less than 10px as discussed in camp.

File: stylesheets/components/snackbar.css (hunk 2 above)
Comment (kedark152) [reply stylesheets/components/snackbar.css]: Sure I will add these changes.

**Comment by kedark152** [comment]
Thank you for your Review!! made some suggested changes and now merging the PR.

**Pull request merged**